workers/unity/Assets/Generated/Improbable.Gdk.Generated.asmdef

```diff
@@ -1,6 +1,14 @@
 {
     "name": "Improbable.Gdk.Generated",
-    "references": [ "Unity.Collections", "Unity.Entities", "Unity.Mathematics", "Unity.Entities.Hybrid", "Improbable.Gdk.Core" ],
+    "references": [ 
+        "Unity.Collections", 
+        "Unity.Entities", 
+        "Unity.Mathematics", 
+        "Unity.Entities.Hybrid",
+        "Improbable.Gdk.Core",
+        "Improbable.Gdk.BuildSystem",
```

Comment:
What happens if someone wants to opt out of the build system? We need modular codegen to make this fully safe

Reply:
that is unfortunately true :(
we could move the build system into Core or have a separeted `GeneratedBuildSystem` folder